client/reader/discover/components/reddit/style.scss

```diff
@@ -38,6 +39,24 @@
 
 	}
 
+	.subscriptions-add-sites__form--container {
+		display: flex;
+		align-items: flex-start;
+		gap: 8px;
+	}
+
+	.subscriptions-add-sites__form-field {
+		flex: 1;
+
+		.components-base-control__field {
+			margin-bottom: 0;
+		}
+	}
+
+	.components-text-control__input {
+		height: 40px;
+	}
+
```

Comment:
These styles are exactly similar to new tab styles. How about we move these in some common place like `/reader/discover/style.scss`?

Reply:
Done 👍